itou/templates/apply/includes/geiq/geiq_diagnosis_details.html

```diff
@@ -1,67 +1,70 @@
 {% if diagnosis.is_valid or job_application.state.is_accepted and diagnosis %}
     {% if diagnosis.allowance_amount %}
         <hr class="my-4">
-        <h3>Éligibilité public prioritaire GEIQ validée</h3>
+        <h3 class="mb-0">Diagnostic public prioritaire GEIQ</h3>
+        <span class="text-success fs-sm">
+            Diagnostic confirmée par {{ diagnosis.author.get_full_name }} ({{ diagnosis.author_structure.display_name }})
```

Comment:
```suggestion
            Diagnostic confirmé par {{ diagnosis.author.get_full_name }} ({{ diagnosis.author_structure.display_name }})
```
Et quelques autres endroits